### PR TITLE
[zh-cn]: update translation of HTTP `Accept-Language` header

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/http/headers/accept-language/index.md [hunk 1]
```diff
@@ -1,50 +1,70 @@
 ---
 title: Accept-Language
 slug: Web/HTTP/Headers/Accept-Language
+l10n:
+  sourceCommit:
```
Comment (T34-active) [inline files/zh-cn/web/http/headers/accept-language/index.md:5]:
```suggestion
  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
```

File: files/zh-cn/web/http/headers/accept-language/index.md [hunk 2]
```diff
@@ -1,50 +1,70 @@
...
 
 ## 指令
 
 - `<language>`
-  - : 用含有两到三个字符的字符串表示的语言码或完整的语言标签。除了语言本身之外，还会包含其他方面的信息，显示在中划线（"-"）后面。最常见的额外信息是国家或地区变种（如"en-US"）或者表示所用的字母系统（如"sr-Lat"）。其他变种诸如拼字法（"de-DE-1996"）等通常不被应用在这种场合。
+  - : 用含有两到三个字符的字符串表示的语言码或完整的语言标签。除了语言本身之外，还会包含其他方面的信息，显示在中划线 `'-'` 后面。最常见的额外信息是国家或地区变种（如 `'en-US'` 或 `'fr-CA'`）或者表示所用的字母系统（如 `'sr-Latn'`）。其他变种诸如拼字法（`'de-DE-1996'`）等通常不被应用在这种场合。
 - `*`
-  - : 任意语言；`"*"` 表示通配符（wildcard）。
+  - : 任意语言；`'*'` 表示通配符（wildcard）。
 - `;q=` (q-factor weighting)
```
Comment (T34-active) [inline files/zh-cn/web/http/headers/accept-language/index.md:53]:
```suggestion
- `;q=` （q-factor 加权）
```

File: files/zh-cn/web/http/headers/accept-language/index.md [hunk 3]
```diff
@@ -1,50 +1,70 @@
...
 ## 指令
 
 - `<language>`
-  - : 用含有两到三个字符的字符串表示的语言码或完整的语言标签。除了语言本身之外，还会包含其他方面的信息，显示在中划线（"-"）后面。最常见的额外信息是国家或地区变种（如"en-US"）或者表示所用的字母系统（如"sr-Lat"）。其他变种诸如拼字法（"de-DE-1996"）等通常不被应用在这种场合。
+  - : 用含有两到三个字符的字符串表示的语言码或完整的语言标签。除了语言本身之外，还会包含其他方面的信息，显示在中划线 `'-'` 后面。最常见的额外信息是国家或地区变种（如 `'en-US'` 或 `'fr-CA'`）或者表示所用的字母系统（如 `'sr-Latn'`）。其他变种诸如拼字法（`'de-DE-1996'`）等通常不被应用在这种场合。
 - `*`
-  - : 任意语言；`"*"` 表示通配符（wildcard）。
-- `;q=` (q-factor weighting)
+  - : 任意语言；`'*'` 表示通配符（wildcard）。
+- `;q=` （q-factor 加权）
```
Comment (T34-active) [inline files/zh-cn/web/http/headers/accept-language/index.md:53]:
```suggestion
- `;q=` （q-factor 权重）
```

File: files/zh-cn/web/http/headers/accept-language/index.md [hunk 4]
```diff
@@ -1,50 +1,70 @@
...
+      <th scope="row">标头类型</th>
+      <td>{{Glossary("Request header", "请求标头")}}</td>
+    </tr>
+    <tr>
+      <th scope="row">{{Glossary("Forbidden header name", "禁止修改的标头")}}</th>
+      <td>无</td>
+    </tr>
+    <tr>
+      <th scope="row">
+        {{Glossary("CORS-safelisted request header", "CORS 安全列出的请求标头")}}
```
Comment (T34-active) [inline files/zh-cn/web/http/headers/accept-language/index.md:28]:
```suggestion
        {{Glossary("CORS-safelisted request header", "列入 CORS 白名单的请求标头")}}
```

File: files/zh-cn/web/http/headers/accept-language/index.md [hunk 5]
```diff
@@ -1,50 +1,70 @@
...
 title: Accept-Language
 slug: Web/HTTP/Headers/Accept-Language
+l10n:
+  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
 ---
 
 {{HTTPSidebar}}
 
-**`Accept-Language`** 请求头允许客户端声明它可以理解的自然语言，以及优先选择的区域方言。借助[内容协商机制](/zh-CN/docs/Web/HTTP/Content_negotiation)，服务器可以从诸多备选项中选择一项进行应用，并使用 {{HTTPHeader("Content-Language")}} 应答头通知客户端它的选择。浏览器会基于其用户界面语言为这个请求头设置合适的值，即便是用户可以进行修改，但是这种情况极少发生（因为可增加指纹独特性，通常也不被鼓励）（译者注：通常只在测试网站的多语言支持时手动修改它；或为进一步减少指纹独特性，改为最常见的英文）。
+**`Accept-Language`** 请求 HTTP 标头表示客户端所偏好的自然语言和区域设置。服务器利用[内容协商](/zh-CN/docs/Web/HTTP/Content_negotiation)机制从这些提议中选出一项，并通过 {{HTTPHeader("Content-Language")}} 响应标头将这一选择告知客户端。浏览器会根据其当前活跃的用户界面语言为该标头设定所需的值。用户很少更改此设置，而且也不建议这样做，因为这可能导致[指纹识别](/zh-CN/docs/Glossary/Fingerprinting).
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/accept-language/index.md:10]:
```suggestion
**`Accept-Language`** 请求 HTTP 标头表示客户端所偏好的自然语言和区域设置。服务器利用[内容协商](/zh-CN/docs/Web/HTTP/Content_negotiation)机制从这些提议中选出一项，并通过 {{HTTPHeader("Content-Language")}} 响应标头将这一选择告知客户端。浏览器会根据其当前活跃的用户界面语言为该标头设定所需的值。用户很少更改此设置，而且也不建议这样做，因为这可能导致[指纹识别](/zh-CN/docs/Glossary/Fingerprinting)。
```

File: files/zh-cn/web/http/headers/accept-language/index.md [hunk 6]
```diff
@@ -1,50 +1,70 @@
...
 
-当服务器无法通过其他方式来确定应当使用的语言时——例如某一特定的 URL，这是用户明确指定的——这个请求头可以用作提示。建议服务器端永远不要覆盖明确指定的信息。`Accept-Language` 消息头的内容通常不在用户的掌控之中（例如在国外旅行时到提供网络服务的场所上网）；另外用户可能会想要浏览非本地用户界面语言的页面。
+当服务器无法通过其他方式（比如使用依赖于用户明确决定的特定 URL）确定目标内容语言时，这个标头可作为提示使用。服务器绝不应覆盖用户的明确语言选择。`Accept-Language` 的内容常常超出用户的控制范围（例如在旅行时）。用户也可能希望访问的语言与用户界面语言不同。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/accept-language/index.md:12]:
```suggestion
当服务器无法通过其他方式（比如使用依赖于用户明确决定的特定 URL）确定目标内容语言时，这个标头可作为提示使用。服务器绝不应覆盖用户的明确语言选择。`Accept-Language` 的内容常常超出用户的控制范围（例如在旅行时）。用户也可能希望访问使用的语言与用户界面并不相同的页面。
```

File: files/zh-cn/web/http/headers/accept-language/index.md [hunk 7]
```diff
@@ -1,50 +1,70 @@
...
 
 ## 语法
 
-```plain
+```http
 Accept-Language: <language>
 Accept-Language: *
 
-// Multiple types, weighted with the {{glossary("quality values", "quality value")}} syntax:
+//使用{{Glossary("Quality Values", "质量价值")}}语法对多个类型进行加权：
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/accept-language/index.md:43]: mdn/content#33297

```suggestion
// 使用质量价值语法对多个类型进行加权：
```

File: files/zh-cn/web/http/headers/accept-language/index.md [hunk 8]
```diff
@@ -58,5 +78,5 @@ Accept-Language: zh-CN,zh;q=0.8,zh-TW;q=0.7,zh-HK;q=0.5,en-US;q=0.3,en;q=0.2
 ## 参见
 
 - HTTP [内容协商](/zh-CN/docs/Web/HTTP/Content_negotiation)
-- 表示此 Header 的内容协商结果的消息头：{{HTTPHeader("Content-Language")}}
-- 其他类似的消息头：{{HTTPHeader("TE")}}、{{HTTPHeader("Accept-Encoding")}}、{{HTTPHeader("Accept-Charset")}}、{{HTTPHeader("Accept")}}
+- 表示此 Header 的内容协商结果的消息标头：{{HTTPHeader("Content-Language")}}
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/accept-language/index.md:81]:
```suggestion
- 表示内容协商结果的标头：{{HTTPHeader("Content-Language")}}
```

File: files/zh-cn/web/http/headers/accept-language/index.md [hunk 9]
```diff
@@ -58,5 +78,5 @@ Accept-Language: zh-CN,zh;q=0.8,zh-TW;q=0.7,zh-HK;q=0.5,en-US;q=0.3,en;q=0.2
...
+- 其他类似的消息标头：{{HTTPHeader("TE")}}、{{HTTPHeader("Accept-Encoding")}}、{{HTTPHeader("Accept")}}
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/accept-language/index.md:82]:
```suggestion
- 其他类似的标头：{{HTTPHeader("TE")}}、{{HTTPHeader("Accept-Encoding")}}、{{HTTPHeader("Accept")}}
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**